Hazelcast.Net/Hazelcast.Client.Spi/ClientPartitionService.cs

```diff
@@ -48,9 +48,22 @@ public ClientPartitionService(HazelcastClient client)
 
         public Address GetPartitionOwner(int partitionId)
         {
-            Address rtn;
-            _partitions.TryGetValue(partitionId, out rtn);
-            return rtn;
+            Address partitionOwner = null;
+            while (_live.Get() && !_partitions.TryGetValue(partitionId, out partitionOwner))
+            {
+                Logger.Finest("Address of a partition cannot be null. Retrying to get it...");
```

Comment:
is the partitionOwner set correctly when the code is here? if not, should we continue?